sdk/cosmosdb/Azure.ResourceManager.CosmosDB/src/autorest.md

```diff
@@ -9,13 +9,18 @@ azure-arm: true
 csharp: true
 library-name: CosmosDB
 namespace: Azure.ResourceManager.CosmosDB
-require: https://github.com/Azure/azure-rest-api-specs/blob/44e83346defd3d4ca99efade8b1ee90c67d9f249/specification/cosmos-db/resource-manager/readme.md
+require: https://github.com/Azure/azure-rest-api-specs/blob/4e7b400ffe0f0d5fabf7b2dcc1e2c65f0be579f5/specification/cosmos-db/resource-manager/readme.md
```

Comment:
The uri must point to the main repo

Reply:
Updated the uri to point to the main repo